content/departments/engineering/product/process/index.md

```diff
@@ -4,43 +4,11 @@ This page contains information that is relevant for how to do well at your job a
 
 ## Product process
```

Comment:
@Joelkw @ryankscott here's the most meaningful change, the rest is links being fixed and minor.